packages/suite-desktop/src/app.ts

```diff
@@ -31,6 +31,9 @@ global.resourcesPath = isDevEnv
     ? path.join(__dirname, '..', 'build', 'static')
     : process.resourcesPath;
 
+// fix bcrypto - use JS implementaion instead of native binaries
+process.env.NODE_BACKEND = 'js';
```

Comment:
any better idea where to place it?

Reply:
I've tried that, but we need to have it runtime. That build script doesn't build bcrypto library, because it is a dependency of utxo-lib, not suite-desktop.

Reply:
it could work in build time (using esbuild) if we start building @trezor/coinjoin https://github.com/trezor/trezor-suite/pull/6542

Reply:
@szymonlesisz @Nodonisko What do you think is a better solution? I tend to prefer built time solution, but I don't see any real difference here.

Reply:
Using JS implementation vs native will have probably significant performance effect. JS is much much slower.

Reply:
JS vs native is a different discussion, I don't know how to rebuild those native binaries for correct architecture (that is the original issue) and the current solution is a workaround (using JS instead of native).

Reply:
I see... anyway I definitely prefer build time solution using esbuild.

Reply:
but the build time solution using esbuild means bundling extra ~ 500 KB of duplicated code (all the dependencies of @trezor/coinjoin)

Reply:
followup issue to find a way how to make the native binaries work also for M1 mac created: https://github.com/trezor/trezor-suite/issues/6558